doc-content/jbpm-docs/src/main/asciidoc/BAM/DashbuilderRuntime-exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -0,0 +1,193 @@
+[id='DashbuilderRuntime-exporting-importing-dashbuilder-data-proc_{context}']
+= Exporting and importing Dashbuilder data
+
+ifdef::PAM,JBPM[]
+
+Dashbuilder is a dashboard and reporting tool integrated in {CENTRAL} and is used by the Datasets editor and Content Manager page. There are three data types:
```

Comment:
Per IBM SG, avoid empty "there is/are", "it is/are", etc. Unclear to users and also a translation issue. So for here, suggest "Dashbuilder supports/contains/displays [whatever] the following data types:"

Reply:
Done